packages/x-charts/src/models/seriesType/pie.ts

```diff
@@ -58,14 +61,16 @@ export interface PieSeriesType<Tdata = PieValueType> extends CommonSeriesType<Td
   arcLabelMinAngle?: number;
   /**
    * The x coordinate of the pie center.
-   * @default width/2 the center of the drawing area.
+   * The '100%' is the width the drawing area.
```

Comment:
Would it also make sense to mention here that this prop can be either number in px as well?
We do it above for `innerRadius` and `outerRadius`. 🤔